stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -476,13 +506,14 @@ function externDecryptAesEcb(byte[] input, byte[] key, handle padding) returns b
 } external;
 
 # Returns AES GCM decrypted value for the given AES GCM encrypted data.
+# ```ballerina byte[]|crypto:Error plainText = crypto:decryptAesGcm(cipherText, key, initVector); ```
 #
 # + input - The content to be decrypted
 # + key - Encryption key
 # + iv - Initialization vector
 # + padding - The padding
 # + tagSize - Tag size
-# + return - Decrypted data or `Error` if key is invalid
+# + return - Decrypted data or `crypto:Error` if key is invalid
```

Comment:
```suggestion
# + return - Decrypted data or `crypto:Error` if key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.